api/v2/views/base.py

```diff
@@ -199,8 +200,9 @@ def perform_update(self, serializer):
             raise exceptions.ParseError(detail=message)
         except Exception as e:
             message = {
-                "An error was encoutered when updating the request."
+                "An error was encoutered when updating the request: %s" % e.message
             }
+            logger.exception(e)
```

Comment:
Will `e.message` include the Exception type information? If we're doing a _catch_ on exception - it might be good to include some info about the exception (though it'll be either a Python lang or DRF exception - give the `import` scope).